Source/JavaScriptCore/API/glib/JSCClass.h.in

```diff
@@ -31,11 +31,12 @@
 G_BEGIN_DECLS
 
 #define JSC_TYPE_CLASS            (jsc_class_get_type())
+#if !ENABLE(2022_GLIB_API)
 #define JSC_CLASS(obj)            (G_TYPE_CHECK_INSTANCE_CAST((obj), JSC_TYPE_CLASS, JSCClass))
 #define JSC_IS_CLASS(obj)         (G_TYPE_CHECK_INSTANCE_TYPE((obj), JSC_TYPE_CLASS))
+#endif
 
-typedef struct _JSCClass JSCClass;
-typedef struct _JSCClassClass JSCClassClass;
+JSC_DECLARE_FINAL_TYPE(JSCClass, jsc_class, JSC, CLASS, GObject)
```

Comment:
Um, you might hate me for this suggestion, but since this is a public header and not the .cpp, there should be a space before the opening paren:

```suggestion
JSC_DECLARE_FINAL_TYPE (JSCClass, jsc_class, JSC, CLASS, GObject)
```

Yeah... in all the public headers....

Reply:
(Nothing that `sed` wouldn't do automatically, BTW)